packages/renderer/src/lib/image/ImagesList.spec.ts

```diff
@@ -60,6 +60,8 @@ beforeEach(() => {
   (window as any).listViewsContributions = listViewsContributionsMock;
   listViewsContributionsMock.mockResolvedValue([]);
   (window as any).getConfigurationValue = vi.fn();
+  (window as any).getConfigurationProperties = vi.fn().mockResolvedValue([]);
+  (window as any).deleteImage = vi.fn().mockResolvedValue(undefined);
```

Comment:
why do we need a resolved value ?

Reply:
1. when getConfigurationProperties got mocked, deleteImage is not a function error appears on stdout
2. when mocking deleteImage with vi.fn().mockResolvedValue() it requires one parameter

I can try if `vi.spyOn(window, 'deleteImage').mockResolvedValue();` works in tests, at least compiler does not complain about it.

Reply:
I mean, if we do only vi.fn(), we're checking the returning promise ?

Reply:
that works too

Reply:
updated